lib/StaticAnalyzer/Core/PlistDiagnostics.cpp

```diff
@@ -169,10 +189,89 @@ static void ReportControlFlow(raw_ostream &o,
     EmitString(o, s) << '\n';
   }
 
+  // Output constraints.
+  EmitConstraints(o, P.getInternalInfos(), indent);
+
   --indent;
   Indent(o, indent) << "</dict>\n";
 }
 
+// TODO: Code just for debugging reasons from here...
```

Comment:
Perhaps, we could put this in a namespace like `debug_hack` or something. So then it would be immediate that the two functions are both for the experiment and no need for the closing comment.